package.json

```diff
@@ -101,21 +97,19 @@
     "ember-wormhole": "^0.5.2",
     "eslint": "^4.19.1",
     "eslint-config-airbnb-base": "^13.0.0",
-    "eslint-plugin-ember": "^6.2.0",
+    "eslint-plugin-ember": "^7.1.0",
     "eslint-plugin-ember-best-practices": "1.1.1",
     "eslint-plugin-import": "^2.12.0",
-    "eslint-plugin-node": "^9.0.1",
+    "eslint-plugin-node": "^10.0.0",
     "fastboot": "2.0.1",
-    "foundation-sites": "6.5.0-rc.4",
-    "husky": "^6.0.0",
+    "foundation-sites": "6.4.1",
     "labs-ember-search": "^3.1.5",
     "labs-ui": "^0.0.24",
-    "lint-staged": ">=10",
```

Comment:
If it's causing huge headaches, I'm cool with deleting this for the foreseeable future but ideally this would be something we remove only temporarily. Hopefully just upgrading to a newer version of `lint-staged` helps.